.copier-answers.image-template.yml

```diff
@@ -0,0 +1,17 @@
+# Changes here will be overwritten by Copier; do NOT edit manually
```

Comment:
I have put a lot of effort in this project to clean the directories from unnecessary files, especially the root directory. Keeping this project clean (i.e. having only needed files) I think is a top priority. I'm being blunt here, sorry, but I cannot accept this as I think this addition is unnecessary and I don't want to see the root directory cluttered with files I deem inappropriate or unnecessary.